src/components/common/TmSessionSignIn.vue

```diff
@@ -182,6 +191,13 @@ export default {
       } else {
         selectedNetwork = selectedNetworksArray[0]
       }
+      // HACK as polkadot addresses don't have a prefix
```

Comment:
I think `selectNetworkByAddress` is a dupe and should be removed in favor of `networkOfAddress`. Possible in another PR.